### ISSUE TITLE
finetuning experiment

### ISSUE THREAD

**Issue by 7nuo** [ngailapdi/SplatTalk-LLaVA-Inference#1]
Dear Author：
when I prepare a small amount dataset and run command "bash scripts/train/finetune_ov.sh",this is the part log of my experiment,  I wonder why the loss is so low and has not changed at all.
"{'loss': 0.0, 'grad_norm': 1.8881510975257518e-07, 'learning_rate': 3.5485766137276894e-06, 'epoch': 0.61}

 61%|██████    | 380/627 [00:48<01:41,  2.44it/s]/home/ubuntu/miniconda3/envs/llava/lib/python3.10/site-packages/torch/utils/checkpoint.py:429: UserWarning: torch.utils.checkpoint: please pass in use_reentrant=True or use_reentrant=False explicitly. The default value of use_reentrant will be updated to be False in the future. To maintain current behavior, pass use_reentrant=True. It is recommended that you use use_reentrant=False. Refer to docs for more details on the differences between the two variants.
  warnings.warn(
/home/ubuntu/miniconda3/envs/llava/lib/python3.10/site-packages/torch/utils/checkpoint.py:61: UserWarning: None of the inputs have requires_grad=True. Gradients will be None
  warnings.warn(
/home/ubuntu/miniconda3/envs/llava/lib/python3.10/site-packages/transformers/models/llava/configuration_llava.py:143: FutureWarning: The `vocab_size` attribute is deprecated and will be removed in v4.42, Please use `text_config.vocab_size` instead.
  warnings.warn(
Step 20 loss: 1.192092824453539e-08
Step 20 loss: 0.0
Step 21 loss: 0.0
Step 21 loss: 1.192092824453539e-08

 61%|██████    | 381/627 [01:03<04:07,  1.00s/it]
                                                 
{'loss': 0.0, 'grad_norm': 9.368045066230518e-08, 'learning_rate': 3.5238730999527683e-06, 'epoch': 0.61}

 61%|██████    | 381/627 [01:03<04:07,  1.00s/it]Step 22 loss: 1.192092824453539e-08Step 22 loss: 1.192092824453539e-08

Step 23 loss: 1.192092824453539e-08
[2025-12-01 20:30:45,293] [WARNING] [stage3.py:2069:step] 1 pytorch allocator cache flushes since last step. this happens when there is high memory pressure and is detrimental to performance. if this is happening frequently consider adjusting settings to reduce memory consumption. If you are unable to make the cache flushes go away consider adding get_accelerator().empty_cache() calls in your training loop to ensure that all ranks flush their caches at the same time
Step 23 loss: 1.192092824453539e-08

 61%|██████    | 382/627 [01:07<04:45,  1.17s/it]
                                                 
{'loss': 0.0, 'grad_norm': 1.898248592903134e-07, 'learning_rate': 3.4992089969837016e-06, 'epoch': 0.61}

 61%|██████    | 382/627 [01:07<04:45,  1.17s/it]Step 24 loss: 1.192092824453539e-08Step 24 loss: 1.192092824453539e-08

[2025-12-01 20:30:49,299] [WARNING] [stage3.py:2069:step] 1 pytorch allocator cache flushes since last step. this happens when there is high memory pressure and is detrimental to performance. if this is happening frequently consider adjusting settings to reduce memory consumption. If you are unable to make the cache flushes go away consider adding get_accelerator().empty_cache() calls in your training loop to ensure that all ranks flush their caches at the same time
Step 25 loss: 0.0
Step 25 loss: 1.192092824453539e-08"
Hope you everything well!

**Comment by 7nuo** [comment]
I noticed that FEAURE_ROOT in /home/ubuntu/SplatTalk-LLaVA-Inference/llava/train/train.py points to /projects/.../FreeSplatLang/output_gaussian_full_100_e75.
Could you explain how this directory is generated and what files it should contain?
